README.md

```diff
@@ -0,0 +1,38 @@
+# nottify-node# Nottify-Node
+
+Mobile music-app's backend that store account's database which serves for login/logout features
+
+# Local run
+
+* First you need to have a MongoDB's database.
+
+* Need an env file to start this app 
+```
+    MONGO_URL = Your MongoDB's url
+    PASS_SEC = Depend on you
+    JWT_SEC = Depend on you
+```
+
+* To run this app locally, clone the project, navigate to the root directory with a command prompt or terminal(where the package.json file is located), and run this below code to download the neccesary dependencies onto your local machine (You can use npm instead of yarn).
+
+```
+    yarn install
+```
+
+
+* Inside the root directory, you can run some built-in commands:
```

Comment:
Vì project của mình chỉ có 1 script để run con BE thôi nên `run some built-in commands` nên đổi thành `run this built-in command to start the local server`